calour/util.py

```diff
@@ -43,6 +43,44 @@
 logger = getLogger(__name__)
 
 
+def _iter_fasta(file_name):
+    '''Iterate over fasta file.
+    Fasta file must contain header line (starting with ">") and one or more sequence lines.
+
+    Parameters
+    ----------
+    file_name: str
+        name of the fasta file
+
+    Yields
+    ------
+    (header, sequence)
+    header: str
+        the header line (without ">")
+    sequence: str
+        the sequence ('ACGT')
+    NOTE: header and sequence are stripped (no new line/spaces at beginning or end)
```

Comment:
change this 2 lines -> the sequence (eg 'ACGT'). Both header and sequence are whitespace stripped.

Reply:
fixed